packages/playground/src/components/contracts_list/contracts_table.vue

```diff
@@ -155,6 +190,22 @@ const props = defineProps({
   },
 });
 
+const rentContracts = ref<NormalizedContract[]>([]);
+
+onMounted(() => {
+  rentContracts.value = props.contracts.value.filter(contract => contract.type === ContractType.RENT);
+  console.log("Rent Contracts: ", rentContracts);
+});
+
+const isNodeInRentContracts = computed(() => {
+  if (contractLocked.value && contractLocked.value.amountLocked === 0) {
+    const nodeIdsInRentContracts = rentContracts.value.map(contract => contract.nodeId);
```

Comment:
why do we create this list of the rent contracts each time we click on a contract?
can't we just do it once after loading the contracts?

Reply:
fixed, I moved it to be calculated on mounting.